packages/mdc-linear-progress/foundation.ts

```diff
@@ -64,11 +65,21 @@ export class MDCLinearProgressFoundation extends MDCFoundation<MDCLinearProgress
 
   setDeterminate(isDeterminate: boolean) {
     this.isDeterminate_ = isDeterminate;
+
     if (this.isDeterminate_) {
       this.adapter_.removeClass(cssClasses.INDETERMINATE_CLASS);
       this.setScale_(this.adapter_.getPrimaryBar(), this.progress_);
       this.setScale_(this.adapter_.getBuffer(), this.buffer_);
     } else {
+      if (this.isReversed_) {
+        // reset translation animation timer that was started when reverse
+        // class was introduced, as to keep it in sync with the new scaling
+        // animation timer that's about to start from adding indeterminate class
```

Comment:
I'd reword to something like this for clarity:

Adding REVERSED_CLASS starts a `translate` animation, while adding INDETERMINATE_CLASS starts a `scale` animation. 
Here, we reset the `translate` animation in order to keep it in sync with the new `scale` animation that will start from adding INDETERMINATE_CLASS below.

Reply:
Reworded. Thanks!!